CHANGELOG.md

```diff
@@ -1,11 +1,39 @@
-Bareos Cookbook CHANGELOG
-==========================
+CHANGELOG
+=========
+Chef-Bareos Cookbook
+--------------------
 
-This file is used to list changes made in each version of the bareos cookbook.
+2.2.13
```

Comment:
I need to add some kind of entry for the current version before this should be merged. I'll give an explanation as to why I did the rework and going forward what the plan may entail...